spec/lcm/integration/user_filters_brick_spec.rb

```diff
@@ -211,17 +214,14 @@ def upload_user_filters_csv(user_filters)
            client_id: 'testingclient'
        },
        {
-           login: 'rubydev+admin@gooddata.com',
+           login: 'iam@promoted.com',
```

Comment:
warning: Style/IndentHash: Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is. (rubocop)